### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by thaihuy711**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
+
+</manifest>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: add end of file

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: remove hardcode

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">Music_16</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml]: Change app name

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
```
Comment (Tuanlvt) [inline app/src/main/res/layout/activity_main.xml]: remove tools

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+</android.support.constraint.ConstraintLayout>
```
Comment (Tuanlvt) [inline app/src/main/res/layout/activity_main.xml]: enter end line

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background" />
+    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+</adaptive-icon>
```
Comment (Tuanlvt) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: enter end line

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background" />
+    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+</adaptive-icon>
```
Comment (Tuanlvt) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: enter end line

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="app_name">Music Player</string>
+
```
Comment (Tuanlvt) [inline app/src/main/res/values/strings.xml]: remove this line

File: app/src/test/java/com/thaihuy/music_16/ExampleUnitTest.java [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() {
+        assertEquals(4, 2 + 2);
+    }
+}
```
Comment (Tuanlvt) [inline app/src/test/java/com/thaihuy/music_16/ExampleUnitTest.java]: enter end line

File: app/build.gradle [hunk 10]
```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.thaihuy.music_16"
```
Comment (Tuanlvt) [inline app/build.gradle]: edit pakage to -> com.framgia.music_16

File: app/app.iml [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+    <orderEntry type="library" name="Gradle: com.android.support:appcompat-v7-27.1.1" level="project" />
+    <orderEntry type="library" scope="TEST" name="Gradle: com.android.support.test.espresso:espresso-idling-resource-3.0.2" level="project" />
+    <orderEntry type="library" name="Gradle: android.arch.lifecycle:livedata-core-1.1.0" level="project" />
+    <orderEntry type="library" scope="TEST" name="Gradle: org.hamcrest:hamcrest-library:1.3@jar" level="project" />
+    <orderEntry type="library" scope="TEST" name="Gradle: org.hamcrest:hamcrest-integration:1.3@jar" level="project" />
+    <orderEntry type="library" name="Gradle: android.arch.core:common:1.1.0@jar" level="project" />
+    <orderEntry type="library" scope="TEST" name="Gradle: net.sf.kxml:kxml2:2.3.0@jar" level="project" />
+    <orderEntry type="library" name="Gradle: android.arch.lifecycle:runtime-1.1.0" level="project" />
+  </component>
+</module>
```
Comment (Tuanlvt) [inline app/app.iml]: enter end line

File: app/src/androidTest/java/com/thaihuy/music_16/ExampleInstrumentedTest.java [hunk 12]
```diff
@@ -0,0 +1,26 @@
+package com.thaihuy.music_16;
```
Comment (Tuanlvt) [inline app/src/androidTest/java/com/thaihuy/music_16/ExampleInstrumentedTest.java]: rename pakage to -> com.framgia.music_16

File: app/src/androidTest/java/com/thaihuy/music_16/ExampleInstrumentedTest.java [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getTargetContext();
+
+        assertEquals("com.thaihuy.music_16", appContext.getPackageName());
```
Comment (Tuanlvt) [inline app/src/androidTest/java/com/thaihuy/music_16/ExampleInstrumentedTest.java]: rename pakage to -> com.framgia.music_16

File: local.properties [hunk 14]
```diff
@@ -0,0 +1,10 @@
+## This file is automatically generated by Android Studio.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file should *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+sdk.dir=/home/huy/Android/Sdk
```
Comment (Tuanlvt) [inline local.properties]: enter end line

**Pull request merged**